File: src/app/domain/purchase-order.ts
import { SearchResultBase } from './search-result-base';

export class Purchase_Order extends SearchResultBase {
  quantity(): number {
    return this.Quantity;
  }
  id(): string {
    return String(this.PurchaseOrderID);
  }
  name(): string {
    return this.MaterialName;
  }
    PurchaseOrderID: number;
    MaterialName: string;
    Quantity: number;    
  }

File: MultiSearchWebAPI/MultSearchWebAPI.Domain/Entities/Purchase_Order.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace MultSearchWebAPI.Domain.Entities
{
    public class Purchase_Order
    {
      public string PurchaseOrderID { get; set; }
      public string MaterialName { get; set; }
      public string Quantity { get; set; }
    }
}
